src/main/webapp/jsp/instructorFeedbackEdit.jsp

```diff
@@ -36,6 +40,7 @@
   </c:if>
    <br>
   <input type="hidden" id="num-questions" value="${fn:length(data.qnForms)}">
+  <input type="hidden" name="${FEEDBACK_SESSION_ENABLE_EDIT}" value="${data.isToBeLoadedInEditMode}">
```

Comment:
@whipermr5 No actually, I was just going by the other examples 😅. Can I place it in a div?